Sources/Models/AvatarPosition.swift

```diff
@@ -73,16 +73,20 @@ public struct AvatarPosition {
     
     // The horizontal position
     public var horizontal: Horizontal
+
+    // The horizontal padding
+    public var padding: HorizontalEdgeInsets
     
     // MARK: - Initializers
     
-    public init(horizontal: Horizontal, vertical: Vertical) {
+    public init(horizontal: Horizontal, vertical: Vertical, padding: HorizontalEdgeInsets = .zero) {
         self.horizontal = horizontal
         self.vertical = vertical
+        self.padding = padding
     }
 
-    public init(vertical: Vertical) {
-        self.init(horizontal: .natural, vertical: vertical)
+    public init(vertical: Vertical, padding: HorizontalEdgeInsets = .zero) {
```

Comment:
Is this the current default in MessageKit? I can't remember

Reply:
Yes I matched the default